### PR TITLE
feat: add hamburgur menu

### LINKED ISSUE #2: added navbar
Here I added navbar, which is responsive.

### PR DESCRIPTION
**Pull request by Bhakti199**
here I added hamburgur menu #2 

Screenshot (1333)

### CODE REVIEW

**Review by JantuDeb — APPROVED** [review]

File: src/Components/HamburgurMenu/HamburgurMenu.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
+.hamburgur-menu {
+  padding: 1rem;
+  position: fixed;
+  z-index: 3;
+  top: 3rem;
+  left: 0;
+  list-style: none;
+  background-color: #eff6ff;
```
Comment (rushikesh3299) [inline src/Components/HamburgurMenu/HamburgurMenu.css]: try to use var for colors.

**Review by rushikesh3299 — APPROVED** [review]
Pretty good that you are generating small pr's

File: src/Components/HamburgurMenu/HamburgurMenu.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        </Link>
+        <Link to="/wishlist-page" className="hamburgur-menu-item">
+          WishList
+        </Link>
+        <Link to="/cart-page" className="hamburgur-menu-item">
+          Cart
+        </Link>
+        <Link to="/" className="hamburgur-menu-item">
+          About us
+        </Link>
```
Comment (SaurabhChawda) [inline src/Components/HamburgurMenu/HamburgurMenu.jsx:25]: Suggestion, Use NavLink instead of Link if You are using ClassName.

**Review by SaurabhChawda — APPROVED** [review]
Design and hamburger Looks good to me, You can Merge it.

**Pull request merged**